osmosis-1/osmosis.zone.json

```diff
@@ -1627,6 +1627,16 @@
       "osmosis_main": false,
       "osmosis_frontier": true,
       "osmosis_info": true
+    },
+    {
+      "chain_name": "comdex",
+      "base_denom": "uharbor",
```

Comment:
there already is an entry for HARBOR earlier in the file. If you could please edit the pool number there instead